src/modules/layout/PageWrapper.tsx

```diff
@@ -103,12 +90,8 @@ const PageWrapper = ({ fullscreen }: PageWrapperProps): JSX.Element => {
           selected={Platform.Player}
           platformsProps={platformProps}
           disabledColor="#999"
-          color={
-            isMobile ? theme.palette.primary.main : theme.palette.secondary.main
-          }
-          accentColor={
-            isMobile ? theme.palette.secondary.main : theme.palette.primary.main
-          }
+          color="#ffffff"
```

Comment:
Should we use a softer black?

Reply:
Not in this place, this black only regulates the small icon that is highlighted for the active view (so the black for the play-icon that is highlighted for the playerview in the viewer-change buttons). I think this should remain aligned with the background of the header. But I would be open to making the overall header a bit less black; if you agree I will put it on my backlog (that's a change in the graasp-ui repo).